frontend/src/js/components/header/DeviceNotifications.tsx

```diff
@@ -11,86 +11,235 @@
-            <a href="https://mender.io/pricing" target="_blank" rel="noopener">
-              mender.io/pricing
-            </a>
-          </p>
-        </>
-      }
+    <div
+      className={`flexbox column margin-bottom-small padding-x-small ${classes.limitContainer} ${warning ? classes.warningBg : ''} ${error ? classes.errorBg : ''}`}
     >
-      {content}
-    </MenderTooltipClickable>
+      <div className="flexbox full-width space-between">
+        <div className="flexbox">
+          <Typography variant="subtitle2" className="capitalized-start">
+            {type}
+          </Typography>
+          {warning && <WarningAmberIcon fontSize="small" color="warning" className="margin-left-x-small" />}
+          {error && <ErrorOutlineIcon fontSize="small" color="error" className="margin-left-x-small" />}
+        </div>
+        <Typography variant="body2">
+          {total.toLocaleString('en-US')}/{limit.toLocaleString('en-US')}
+        </Typography>
+      </div>
+      <div className="margin-bottom-x-small margin-top-x-small">
+        <LinearProgress color={color} variant="determinate" value={Math.floor((total / limit) * 100)} />
+      </div>
+      <div className="flexbox">
+        <Typography variant="caption">{Math.floor((total / limit) * 100)}% used</Typography>
+        {warning && (
+          <>
+            <Typography variant="caption" className="margin-left-x-small margin-right-x-small">
+              •
+            </Typography>{' '}
+            <Typography variant="caption" color="warning">
+              Near limit
+            </Typography>
+          </>
+        )}
+        {error && (
+          <>
+            <Typography variant="caption" className="margin-left-x-small margin-right-x-small">
+              •
+            </Typography>{' '}
+            <Typography variant="caption" color="error">
+              Limit reached
+            </Typography>
+          </>
+        )}
+      </div>
+    </div>
+  );
+};
+
+const DeviceNotifications = ({ className = '', total, pending }) => {
+  const { classes } = useStyles();
+  const navigate = useNavigate();
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const [tooltip, setTooltip] = useState(false);
+  const handleOpen = (event: MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+    setTooltip(false);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  const onManageDeviceLimit = () => {
+    handleClose();
+    navigate('/settings/subscription');
+  };
+  const viewPendingClick = () => {
+    handleClose();
+    navigate('/devices/pending');
+  };
+  const { counts: accepted } = useSelector(getAcceptedDevices);
+  const { isAdmin } = useSelector(getUserRoles);
+
+  const limits = useSelector(getDeviceLimits);
+  const mappedLimits = Object.entries(limits)
+    .filter(([, limit]) => limit > 0)
```

Comment:
I think this would at least need 
```suggestion
    .filter(([, limit]) => limit !== 0)
```
to allow us to show unlimited tiers (with a limit of `-1`), or more likely some additional logic, as the `standard` devices are currently still unlimited for both -1 and 0 😬...

Reply:
looks like that wasn't included in the task design (only one of the related ones: https://www.figma.com/design/ZTGKMcKtHrZ0V7pfdws7KM/Zephyr-UX-changes?node-id=4524-2919&t=qSkPRQsL0cfGSUT4-4) 😞 - it will be present in on-prem setups though

Reply:
I've added the commit that treats -1 as unlimited, but if we need to handle both the old and new approaches to 0, I think it deserves a separate task. We can wait with merging this one

Reply:
hmmm I _think_ we kept the `standard w/ limit 0 == unlimited` mainly due to the potential problems for the UI, but I'm not sure... I guess since -1 is _the future_ I would rather merge it and try to align the rest of the product to the unified understanding, then to pour more complexity to cover this up 🫣...